importers/rules.go

```diff
@@ -0,0 +1,256 @@
+// Copyright 2020 The Prometheus Authors
```

Comment:
I think this promtool-specific code should go beside promtool, not clutter up the top-level directory.

Reply:
i moved it to cmd/promtool/rules.go